kubernetes/mock/mock.go

```diff
@@ -28,12 +28,12 @@ func (o *ClientMock) GetDashboards(namespace string) ([]v1alpha1.MonitoringDashb
 
 func FakeChart(id, dataType string) v1alpha1.MonitoringDashboardChart {
 	return v1alpha1.MonitoringDashboardChart{
-		Name:       "My chart " + id,
-		Unit:       "s",
-		UnitScale:  10.0,
-		Spans:      6,
-		MetricName: "my_metric_" + id,
-		DataType:   dataType,
+		Name:      "My chart " + id,
+		Unit:      "s",
+		UnitScale: 10.0,
+		Spans:     6,
+		Metrics:   []v1alpha1.MonitoringDashboardMetric{{DisplayName: "My chart " + id, MetricName: "my_metric_" + id}},
```

Comment:
Here I see the "MetricName" is not used on MonitoringDashboardChart.
Is that intended ? I mean, perhaps it was left intentional for some compability ?

That trigger is on this case we would need a v1alpha2 or v1beta1 then, not sure, I don't *want* to enter in that versioning but I'm 50/50 seeing changes in the model on that sense.

Reply:
I've just updated the mock to use the new model and not `MetricName` which I deprecate.
I don't think it's worth it to bump versioning, as long as we keep backward compatibility, old dashboards continue to work and we don't have to go into the trouble of versioning *right now*.